src/deployment/deploy.py

```diff
@@ -465,12 +465,13 @@ def setup_rbac(self) -> None:
                     subscription=self.get_subscription_id(),
                 )
 
-            assign_instance_app_role(
-                self.application_name,
-                onefuzz_cli_app["displayName"],
-                self.get_subscription_id(),
-                OnefuzzAppRole.ManagedNode,
-            )
+            if not self.skip_aad_setup:
+                assign_instance_app_role(
+                    self.application_name,
+                    onefuzz_cli_app["displayName"],
+                    self.get_subscription_id(),
+                    OnefuzzAppRole.ManagedNode,
```

Comment:
the role here should be `CliClient.CliClient`

Reply:
I didn't actually update the role there, just added the condition. I'm not sure how long it's been ManagedNode, but I wanted to be clear that there was no change in this PR.

Reply:
It should be changed then. We are currently assigning the wrong role to the cli application. We are not enforcing this currently but when we start implementing the access control it might prevent the cli from accessing the service. 
If you don't want to address it in this PR, you can open an issue to fix it later.

Reply:
I'll address here!